flink-kubernetes/src/main/java/org/apache/flink/kubernetes/highavailability/KubernetesLeaderElectionDriver.java

```diff
@@ -89,11 +85,6 @@ public KubernetesLeaderElectionDriver(
                 kubeClient.createLeaderElector(
                         leaderElectionConfiguration, new LeaderCallbackHandlerImpl());
 
-        this.configMapLabels =
-                KubernetesUtils.getConfigMapLabels(
```

Comment:
getConfigMapLabels sets some labels that are marked for deprecation (https://github.com/apache/flink/commit/b7e8b792c086c3c445ee8429fbcfe035097a878c). The intention was to remove the labels because Flink would internally rely on the ConfigMap name and, therefore, reduce the k8s server API calls (see FLINK-33598).

We might want to revert the https://github.com/apache/flink/commit/b7e8b792c086c3c445ee8429fbcfe035097a878c if we decide that it makes sense to keep it even if Flink is not using it internally. Instead, we should mark this as @Public API and document it if we want these labels to be still usable by users for external cleanup of k8s resources.

@Myasuka WDYT? Reverting `b7e8b792` should be fine since it was just introduced in 1.19.

Reply:
Only some labels are deprecated, not all of them (like app/type).
These 2 in particular aren't very useful because they are static values. I don't see a big use-case for them.

Reply:
ah true. :facepalm: I mixed up the variable names. The `clusterId` is covered by the `app` label. :+1: you're right.